api/controllers/UserController.js

```diff
@@ -19,7 +19,12 @@ module.exports = {
       } else {
         var updatedUser = {};
         if (req.params.intro !== undefined) {
-          updatedUser.intro = req.params.intro;
+          if (req.params.intro.length > 10000) {
```

Comment:
Yeah, you need to have this incase someone manually edits the javascript client side, but the error itself doesn't massively matter because if a user doesn't understand the error they get back after hacking the client side javascript, we don't care.